app/controllers/bets_controller.rb

```diff
@@ -1,9 +1,13 @@
 class BetsController < ApplicationController
+  include Utilities::Paginator
+  helper_method :paginator
```

Comment:
Might be able to just call `helper Utilities::Paginator` if we make paginator the only public api for the Paginator module. 
Not 100% sure thats the right move...